rhobs/alerting/data_plane/prometheus.high_memory_usage_alert.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-high-memory-usage-alert-rule
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: High_Memory_Usage_Alert
+    interval: 5m
+    rules:
+    - alert: High_Memory_Usage_Alert
```

```suggestion
    - alert: HighMemoryUsageAlert
```
No need for underscores with using CamelCases